CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.7.0
```

Comment:
But it also requires that the `GOVUK_APP_DOMAIN` variable is set by all apps that use this gem.